city_scrapers/spiders/chi_infrastructure_trust.py

```diff
@@ -0,0 +1,115 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiInfrastructureTrustSpider(CityScrapersSpider):
+    name = "chi_infrastructure_trust"
+    agency = "Chicago Infrastructure Trust"
+    timezone = "America/Chicago"
+    allowed_domains = ["chicagoinfrastructure.org"]
+    start_urls = [
+        "http://chicagoinfrastructure.org/public-records/meeting-records-2/",
+        "http://chicagoinfrastructure.org/public-records/scheduled-meetings/"
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        if response.url == self.start_urls[0]:
+            for item in response.css(".entry").xpath('.//p'):
+                if item.xpath('.//span'):
+                    continue
+
+                yield self._generate_meeting(item, response)
+
+        elif response.url == self.start_urls[1]:
+            item = response.css(".entry").xpath('.//p')[1]
+
+            yield self._generate_meeting(item, response)
+
+    def _generate_meeting(self, item, response):
+        """Generate the meeting object"""
+        meeting = Meeting(
+            title=self._parse_title(item),
+            description=self._parse_description(item),
+            classification=self._parse_classification(item),
+            start=self._parse_start(item, response),
+            end=self._parse_end(item),
+            all_day=self._parse_all_day(item),
+            time_notes=self._parse_time_notes(item),
+            location=self._parse_location(item),
+            links=self._parse_links(item),
+            source=self._parse_source(response),
+        )
+
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+
+        return meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "Board Meeting"
```

Comment:
Just a tiny change here, since "meeting" is redundant so if we're hard-coding it we might as well remove it 

```suggestion
        return "Board of Directors"
```